### DISCUSSION TITLE
[Bug]: docker compose version is not running saying Your cache folder contains root-owned files, due to a bug in

### DISCUSSION THREAD

**Discussion by dims12** [danny-avila/LibreChat#2151] [Troubleshooting]
### What happened?

The following error is displayed by docker compose:

> LibreChat         |
> LibreChat         | > LibreChat@0.6.10 backend
> LibreChat         | > cross-env NODE_ENV=production node api/server/index.js
> LibreChat         |
> LibreChat         | npm ERR! code EACCES
> LibreChat         | npm ERR! syscall mkdir
> LibreChat         | npm ERR! path /.npm
> LibreChat         | npm ERR! errno -13
> LibreChat         | npm ERR!
> LibreChat         | npm ERR! Your cache folder contains root-owned files, due to a bug in
> LibreChat         | npm ERR! previous versions of npm which has since been addressed.
> LibreChat         | npm ERR!
> LibreChat         | npm ERR! To permanently fix this problem, please run:
> LibreChat         | npm ERR!   sudo chown -R 1001:1002 "/.npm"
> LibreChat         |
> LibreChat         | npm ERR! Log files were not written due to an error writing to the directory: /.npm/_logs
> LibreChat         | npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal

### Steps to Reproduce

Just follow the instructions from Docker Compose Installation Guide page

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I've never seen this issue, were you still able to get it running correctly? Could be due to these values in the compose file:

>     user: "${UID}:${GID}"

I'll try adjusting the Dockerfile slightly and see if the issue can be resolved
